[2023-12-19T05:04:01.105+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_and_cc_complaints-v01.remove_replaced_dim_cols.remove_original_issue_in_fact_table scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T05:04:01.110+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_and_cc_complaints-v01.remove_replaced_dim_cols.remove_original_issue_in_fact_table scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T05:04:01.112+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-19T05:04:01.127+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): remove_original_issue_in_fact_table> on 2023-12-01 00:00:00+00:00
[2023-12-19T05:04:01.133+0000] {standard_task_runner.py:57} INFO - Started process 1384 to run task
[2023-12-19T05:04:01.138+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bank_and_cc_complaints-v01.remove_replaced_dim_cols', 'remove_original_issue_in_fact_table', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/bank_and_cc_complaints.py', '--cfg-path', '/tmp/tmphkv2jmz0']
[2023-12-19T05:04:01.141+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask remove_original_issue_in_fact_table
[2023-12-19T05:04:01.199+0000] {task_command.py:416} INFO - Running <TaskInstance: bank_and_cc_complaints-v01.remove_replaced_dim_cols.remove_original_issue_in_fact_table scheduled__2023-12-01T00:00:00+00:00 [running]> on host 2ee4b7dc3cc7
[2023-12-19T05:04:01.267+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dunnbc22' AIRFLOW_CTX_DAG_ID='bank_and_cc_complaints-v01.remove_replaced_dim_cols' AIRFLOW_CTX_TASK_ID='remove_original_issue_in_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2023-12-19T05:04:01.270+0000] {sql.py:274} INFO - Executing: -- table_name, column_name
ALTER TABLE ***_complaints
DROP COLUMN issue;
[2023-12-19T05:04:01.286+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T05:04:01.299+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T05:04:01.310+0000] {sql.py:418} INFO - Running statement: -- table_name, column_name
ALTER TABLE ***_complaints
DROP COLUMN issue;, parameters: None
[2023-12-19T05:04:01.326+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=bank_and_cc_complaints-v01.remove_replaced_dim_cols, task_id=remove_original_issue_in_fact_table, execution_date=20231201T000000, start_date=20231219T050401, end_date=20231219T050401
[2023-12-19T05:04:01.350+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-19T05:04:01.373+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
